### PR TITLE
Uiuxupdates

### PR DESCRIPTION
**Pull request by SohamDutta1216**
UI/UX Updates 

- SCSS and SASS Added 
- some cleanup on folder structuring for scss and front-end components (still need to do more) 
- complete revamp of UI colors / logo and other styles

### CODE REVIEW

**Review by marvinmarnold — COMMENTED** [review]
Hey, thanks for this work. I think things look better now for sure. Some small comments:

- Submit button not styled on feedback page
Screenshot 2024-01-02 at 12 28 21 PM
- Feedback page: tabs look a little weird and `Please enter your name` hard to read.
Screenshot 2024-01-02 at 12 28 58 PM
- There are merge conflicts now
- Can you create a version of the logo without "SAWT" so that I can create a favicon for it? Ideally you could update the favicon too. 
- I'm not sure about the blue in the logo. Maybe it's making it too noisy?

**Comment by SohamDutta1216** [comment]
> Hey, thanks for this work. I think things look better now for sure. Some small comments:
> 
> * Submit button not styled on feedback page
>   Screenshot 2024-01-02 at 12 28 21 PM
> * Feedback page: tabs look a little weird and `Please enter your name` hard to read.
>   Screenshot 2024-01-02 at 12 28 58 PM
> * There are merge conflicts now
> * Can you create a version of the logo without "SAWT" so that I can create a favicon for it? Ideally you could update the favicon too.
> * I'm not sure about the blue in the logo. Maybe it's making it too noisy?

Fixed up everything you mentioned as well as pulled and merged the newest file. Let me know if anything else is needed !

File: packages/web/app/hooks/useLocalStorage.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import {
```
Comment (marvinmarnold) [inline packages/web/app/hooks/useLocalStorage.tsx:1]: Can you explain what this file is doing? I don't see it referenced anywhere.

**Review by marvinmarnold — COMMENTED** [review]
Awesome. Real close. Some additional comments.

- Clear button looks thin
Screenshot 2024-01-07 at 10 48 59 AM

- Tabs still look a little off. IMHO, the bottom half of the tabs shouldn't show curvature.  
Screenshot 2024-01-07 at 10 50 34 AM

- The logo and color scheme is great for now. Longer term, we'll need to talk with the whole Eye on Surveillance group and decide how/if we want to merge color/logo with EOS branding.

**Comment by ayyubibrahimi** [comment]
The card color changes once it's clicked. Can we have this card color match the card color before it's clicked? 

Before:
Screenshot 2024-01-10 at 5 14 14 PM

After:
Screenshot 2024-01-10 at 5 14 33 PM

**Comment by ayyubibrahimi** [comment]
Most up-to-date version of Sawt has "beta" as subscript. Thoughts about the best way to integrate a "beta" marker? 

Screenshot 2024-01-10 at 7 43 56 PM

**Comment by SohamDutta1216** [comment]
> Most up-to-date version of Sawt has "beta" as subscript. Thoughts about the best way to integrate a "beta" marker?
> 
> Screenshot 2024-01-10 at 7 43 56 PM

added a beta via html and css so u can move it around easily, LMK if this works !

**Comment by SohamDutta1216** [comment]
> The card color changes once it's clicked. Can we have this card color match the card color before it's clicked?
> 
> Before: Screenshot 2024-01-10 at 5 14 14 PM
> 
> After: Screenshot 2024-01-10 at 5 14 33 PM

fixed this !

File: packages/web/components/HomeResults/HomeResults.tsx [hunk 2]
```diff
@@ -45,33 +45,34 @@ export default function HomeBanner() {
       });
     }
   }, [isInView, isLoading, fetchMoreCards, hasMoreCards]);
-
+  console.log(cards);
```
[the remark this answers is not in the document]
Comment (SohamDutta1216) [reply packages/web/components/HomeResults/HomeResults.tsx]: deleted this !

File: packages/web/app/hooks/useLocalStorage.tsx (hunk 1 above)
Comment (SohamDutta1216) [reply packages/web/app/hooks/useLocalStorage.tsx]: was testing something with it but deleted it !

**Review by marvinmarnold — APPROVED** [review]
thanks lgtm

**Pull request merged**